### ISSUE TITLE
Syntax out of syntax with latest StandardRb rules

### ISSUE THREAD

**Issue by D-system** [Data-Liberation-Front/csvlint.rb#284]
> Please provide a general summary of the issue in the Issue Title above
> fill out the headings below as applicable to the issue you are reporting,
> deleting as appropriate but offering us as much detail as you can to help us resolve the issue

### Problem
PRs can not be merge due to syntax.

https://github.com/Data-Liberation-Front/csvlint.rb/actions/runs/7325235579/job/19985406230?pr=281
```
Run bundle exec standardrb
WARNING: this project is being migrated to standard gradually via `/home/runner/work/csvlint.rb/csvlint.rb/.standard_todo.yml` and is ignoring these files:
  features/step_definitions/validation_errors_steps.rb
  features/support/load_tests.rb
  lib/csvlint/cli.rb
  lib/csvlint/csvw/column.rb
  lib/csvlint/csvw/date_format.rb
  lib/csvlint/csvw/number_format.rb
  lib/csvlint/csvw/property_checker.rb
  lib/csvlint/csvw/table_group.rb
  lib/csvlint/field.rb
  lib/csvlint/schema.rb
  lib/csvlint/validate.rb
  spec/validator_spec.rb
standard: Use Ruby Standard Style (https://github.com/standardrb/standard)
  lib/csvlint/csvw/column.rb:129:13: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/column.rb:136:13: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/column.rb:259:[4](https://github.com/Data-Liberation-Front/csvlint.rb/actions/runs/7325235579/job/19985406230?pr=281#step:5:5)[5](https://github.com/Data-Liberation-Front/csvlint.rb/actions/runs/7325235579/job/19985406230?pr=281#step:5:6): Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/column.rb:2[6](https://github.com/Data-Liberation-Front/csvlint.rb/actions/runs/7325235579/job/19985406230?pr=281#step:5:7)0:51: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/column.rb:266:9: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/column.rb:284:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/column.rb:294:11: Style/RedundantReturn: Redundant `return` detected.
  lib/csvlint/csvw/column.rb:300:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/column.rb:306:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/column.rb:312:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/column.rb:318:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/column.rb:324:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/column.rb:330:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/column.rb:336:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/column.rb:342:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/column.rb:348:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/column.rb:354:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/column.rb:360:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/column.rb:366:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/column.rb:3[7](https://github.com/Data-Liberation-Front/csvlint.rb/actions/runs/7325235579/job/19985406230?pr=281#step:5:8)2:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:63:17: Style/UnlessLogicalOperators: Do not use mixed logical operators in an `unless`.
  lib/csvlint/csvw/property_checker.rb:102:25: Lint/SymbolConversion: Unnecessary symbol conversion; use `:":invalid_#{property}"` instead.
  lib/csvlint/csvw/property_checker.rb:119:13: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:126:13: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:133:13: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:140:13: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:147:13: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:155:13: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:162:13: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:170:15: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:1[8](https://github.com/Data-Liberation-Front/csvlint.rb/actions/runs/7325235579/job/19985406230?pr=281#step:5:9)0:15: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:200:15: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:202:15: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:20[9](https://github.com/Data-Liberation-Front/csvlint.rb/actions/runs/7325235579/job/19985406230?pr=281#step:5:10):13: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:229:[11](https://github.com/Data-Liberation-Front/csvlint.rb/actions/runs/7325235579/job/19985406230?pr=281#step:5:12): Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:252:[13](https://github.com/Data-Liberation-Front/csvlint.rb/actions/runs/7325235579/job/19985406230?pr=281#step:5:14): Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:254:13: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:261:13: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:272:13: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:274:13: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:280:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:390:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:400:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:407:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:426:21: Style/UnlessLogicalOperators: Do not use mixed logical operators in an `unless`.
  lib/csvlint/csvw/property_checker.rb:441:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:446:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:486:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:495:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:511:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:5[14](https://github.com/Data-Liberation-Front/csvlint.rb/actions/runs/7325235579/job/19985406230?pr=281#step:5:15):55: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:543:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:547:60: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:548:60: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:549:54: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:575:13: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:581:56: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/csvw/property_checker.rb:583:11: Style/RedundantReturn: Redundant `return` detected. To return multiple values, use an array.
  lib/csvlint/field.rb:[19](https://github.com/Data-Liberation-Front/csvlint.rb/actions/runs/7325235579/job/19985406230?pr=281#step:5:20):40: Style/RedundantParentheses: Don't use parentheses around a logical expression.
  lib/csvlint/schema.rb:[39](https://github.com/Data-Liberation-Front/csvlint.rb/actions/runs/7325235579/job/19985406230?pr=281#step:5:40):7: Lint/UselessRescue: Useless `rescue` detected.
  lib/csvlint/validate.rb:500:7: Lint/UselessRescue: Useless `rescue` detected.
  lib/csvlint/validate.rb:593:[48](https://github.com/Data-Liberation-Front/csvlint.rb/actions/runs/7325235579/job/19985406230?pr=281#step:5:49): Style/StringLiteralsInInterpolation: Prefer double-quoted strings inside interpolations.
  lib/csvlint/validate.rb:594:63: Style/StringLiteralsInInterpolation: Prefer double-quoted strings inside interpolations.
  lib/csvlint/validate.rb:595:62: Style/StringLiteralsInInterpolation: Prefer double-quoted strings inside interpolations.
  lib/csvlint/validate.rb:599:[52](https://github.com/Data-Liberation-Front/csvlint.rb/actions/runs/7325235579/job/19985406230?pr=281#step:5:53): Style/StringLiteralsInInterpolation: Prefer double-quoted strings inside interpolations.
  lib/csvlint/validate.rb:[60](https://github.com/Data-Liberation-Front/csvlint.rb/actions/runs/7325235579/job/19985406230?pr=281#step:5:61)0:63: Style/StringLiteralsInInterpolation: Prefer double-quoted strings inside interpolations.
standard: Run `standardrb --fix` to fix up to [62](https://github.com/Data-Liberation-Front/csvlint.rb/actions/runs/7325235579/job/19985406230?pr=281#step:5:63) problems.
```

### Desired Behaviour (for improvement suggestions only)
Update syntax

### Steps to Reproduce (for problems)
1. `bundle exec standardrb`
